Python/QR_code_generator/Readme.md

```diff
@@ -0,0 +1,26 @@
+# QR Code Generator
+
+## Script generates QR code for data listed in a csv file
+
+How to use
+
+```
+python qrcode_generator.py --f <csv input file>
+```
+
+Example:
+
+```
+python qrcode_generator.py --f input.csv
+```
+
+### Format of input file
+
+extension: `.csv`<br>
```

Comment:
Format this part properly for a more visual appeal

Reply:
Addressed changes. Let me know if this works.